app/assets/javascripts/mapping.js

```diff
@@ -69,16 +28,24 @@ var MRConfig = (function () {
 (function() {
     var map;
     $(document).ready(function() {
-        var road_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/map/{z}/{x}/{y}.png', {
```

Comment:
I doesn't look like it is in 0.7.7 at least, but I leveraged the idea which will apply in the next PR.